invofi/apps/frontend/src/lib/migrations/002_securitization.sql

```
@@ -0,0 +1,177 @@
+  on fractional_positions for insert
+  with check (lender_id = auth.uid());
+
+create policy "Lender can update own position"
+  on fractional_positions for update
+  using (lender_id = auth.uid());
+
+-- Price history: world-readable, system-written (service role) or originator
+create policy "Anyone can read price history"
+  on price_history for select using (true);
+
+create policy "Authenticated users can insert price history"
```

SQLFluff (4.2.2)

[error] Do not use special characters in identifiers.

(RF05)